File: erp-app/src/app/modules/setup/models/company-search-dto.ts
import { PagingDTO } from "src/app/shared/models/paging-dto";

export class CompanySearchDTO extends PagingDTO {
    id: number;
    isActive:boolean;
    name: string;
    contactPerson: string;
    contactTelephone: string;
    websiteLink: string;
    addressDetails: string;
}

File: ERPServer/Shared/Shared/Entities/Setup/CompanySearchDTO.cs
using Entities.Account;
using Entities.Shared;
using Shared.Entities.Shared;
using Shared.Enums;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Diagnostics.Metrics;

namespace Shared.Entities.Setup
{
    public class CompanySearchDTO :Paging
    {
        public string LogoURL { get; set; }
        public string LogoBase64 { get; set; }
        public string Name { get; set; }
        public string ContactPerson { get; set; }
        public string ContactTelephone { get; set; }
        public string WebsiteLink { get; set; }
        public string AddressDetails { get; set; }

    }
}
